artifacts/CAMARA_common.yaml

```diff
@@ -356,26 +494,52 @@ components:
       content:
         application/json:
           schema:
-            $ref: "#/components/schemas/ErrorInfo"
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 422
+                  code:
+                    enum:
+                      - IDENTIFIER_MISMATCH
+                      - SERVICE_NOT_APPLICABLE
+                      - UNIDENTIFIABLE_IDENTIFIER

```

Comment:
The proposal in [this](https://github.com/camaraproject/Commonalities/issues/321#issuecomment-2429158445) comment was `MISSING_IDENTIFIER`, which I think is more intuitive (and better English!) than `UNIDENTIFIABLE_IDENTIFIER`.

So my preference is to use `MISSING_IDENTIFIER` if there is no strong reason to use `UNIDENTIFIABLE_IDENTIFIER`

Reply:
Agree, I will update to MISSING_IDENTIFIER

Reply:
Done